docs/v1/use-cases.md

```diff
@@ -1,9 +1,26 @@
 ## V1 Use Cases
 
-This document contains the complete list of accepted use-cases for the v1 version of `service-catalog`.
+This document contains the complete list of accepted use-cases for the v1
+version of `service-catalog`.
 
-## Use Cases
+## Use cases
 
-The v1 service-catalog supports the following use cases:
+1.  Sharing services:
+    1.  As a SaaS provider that already runs a Service Broker, I want users of
+        Kubernetes to be able to use my service via the Service Broker API, so
+        that I can grow my user base to include users of Kubernetes
 
-1. <TBD>
+### SaaS providers with existing Service Brokers
+
+There are numerous SaaS providers that already operate Service Brokers today.
+It should be possible for the operator of an existing Service Broker to
+publish their services into the catalog and have them consumed by users of
+Kubernetes.  This offers a new set of users to the service operator and offers
+a wide variety of SaaS products to users of Kubernetes.
+
+When a Kubernetes user binds to a service of this type, the catalog creates
+the following resources in their namespace:
+
+1.  A Kubernetes `Service` instance, to be a stable network endpoint that makes
```

Comment:
Ah, you could be right and I misread it.  Use of the term "service instance" could be problematic since that means something very concrete to the CF SB, which is how I thought it was being used.  Perhaps we should use the phrasing you used instead. 

With that being said... I do kind of wonder if this will be possible since the credentials returned by the SB may be very service specific and I'm not sure we can always extract the host/IP from them to setup a Kube endpoint. If the IP is external, do we really need to define a Kube endpoint? Can't the app just access it directly via the endpoint provided in the credentials? What's the value of the extra layer?

Should we really be talking about it at this level anyway? At the use-case level we should probably avoid discussing solutions.